README.md

```diff
@@ -15,26 +15,25 @@ cargo build --release
 ```
 
 ## Running
-Chombot requires a few settings passed via environment variables. These are:
-* `DISCORD_TOKEN` - the Discord bot token. You can obtain one by [creating a Discord app](https://discord.com/developers/applications). Make sure to copy your Bot token, not a client secret.
-* `GUILD_ID` - your guild ID.
-* `FEATURE_TOURNAMENTS_WATCHER` - `true`, if you want to receive the notification about EMA tournament updates.
-* `TOURNAMENTS_WATCHER_CHANNEL_ID` - ID of the channel used for notifications about EMA tournament updates.
+Chombot requires a few settings passed via the `config.toml` file that must be present in the working directory. Note that this file serves as a persistence storage for the bot's state, so the bot process must have write permissions for it. These are:
+
+* `discord_token` - the Discord bot token. You can obtain one by [creating a Discord app](https://discord.com/developers/applications). Make sure to copy your Bot token, not a client secret.
 
 Example:
 
 ```shell
-export DISCORD_TOKEN=yourdiscordtoken
-export GUILD_ID=12345
-export FEATURE_TOURNAMENTS_WATCHER=true
-export TOURNAMENTS_WATCHER_CHANNEL_ID=98765
+echo 'discord_token="2137"' > config.toml
 cargo run --bin chombot --release
 ```
 
 ### chombot-kcc
 In addition to the base version of Chombot, there is also an enhanced version called Chombot-kcc available. This is version tailored for the needs of Krakow Chombo Club and contains a few additional (possibly hermetic) features.
 
-This version requires a few config values in addition to the ones described above. Those are:
+Chombot-kcc requires some config values defined as environment variables, as opposed to the `config.toml` file approach for the base Chombot version. Those are:
```

Comment:
I think having the same configuration method for both bot versions might be easier in the long term for maintainability and ease of use.